### PR TITLE
[13.0][FIX]purchase_blanket_order: check_date_planned

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**

### CODE REVIEW

**Review by JordiMForgeFlow — APPROVED** [review]
Makes sense 👍🏼

File: purchase_blanket_order/models/purchase_order.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def onchange_blanket_order_line(self):
     @api.constrains("date_planned")
     def check_date_planned(self):
         for line in self:
-            date_planned = fields.Date.from_string(self.date_planned)
+            date_planned = fields.Date.from_string(line.date_planned)
```
Comment (MiquelRForgeFlow) [inline purchase_blanket_order/models/purchase_order.py:159]:
```suggestion
            date_planned = fields.Date.from_string(line.date_planned or self.date_planned)
```

**Review by MiquelRForgeFlow — COMMENTED** [review]
Makes sense.

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**